otel-trace/config-todo-service-otel-prod.properties

```diff
@@ -0,0 +1,4 @@
+otel.service.name=todo-service
```

Comment:
`otel.service.name=todo-service-otel` is better to distinguish OTEL SDK traces from Catchpoint Tracing SDK traces